src/app/options.cpp

```diff
@@ -312,6 +434,27 @@ QBtCommandLineParameters parseCommandLine(const QStringList &args)
             else if (arg == CONFIGURATION_OPTION) {
                 result.configurationName = CONFIGURATION_OPTION.value(arg);
             }
+            else if (arg == SAVE_PATH_OPTION) {
+                result.savePath = SAVE_PATH_OPTION.value(arg);
+            }
+            else if (arg == PAUSED_OPTION) {
+                result.addPaused = PAUSED_OPTION.value(arg);
```

Comment:
So, continuing, my idea is: only `result.addPaused` has to be of type `TriStateBool`. The option itself is a bool option which accepts value and can be omitted. The third case is here already, in this very test.